From 6a60d6c90257529fc0610b1fde9c8f6f272dce4e Mon Sep 17 00:00:00 2001
From: RekGRpth <rekgrpth@gmail.com>
Date: Fri, 26 Apr 2019 10:14:08 +0500
Subject: do not configure_remote_session


diff --git a/contrib/postgres_fdw/connection.c b/contrib/postgres_fdw/connection.c
index fe4893a8e0..66a94f5c62 100644
--- a/contrib/postgres_fdw/connection.c
+++ b/contrib/postgres_fdw/connection.c
@@ -284,7 +284,7 @@ connect_pg_server(ForeignServer *server, UserMapping *user)
 					 errhint("Target server's authentication method must be changed.")));
 
 		/* Prepare new session for use */
-		configure_remote_session(conn);
+//		configure_remote_session(conn);
 
 		pfree(keywords);
 		pfree(values);
